include/boost/geometry/strategies/geographic/buffer_point_circle.hpp

```diff
@@ -74,51 +74,48 @@ public :
                 DistanceStrategy const& distance_strategy,
                 OutputRange& output_range) const
     {
-        typedef typename boost::range_value<OutputRange>::type output_point_type;
+        using output_point_type = typename boost::range_value<OutputRange>::type;
 
-        typedef typename select_calculation_type
+        using calculation_type = typename select_calculation_type
             <
                 Point, output_point_type,
                 CalculationType
                 //double
-            >::type calculation_type;
+            >::type;
+
+        auto const lon_rad = get_as_radian<0>(point);
+        auto const lat_rad = get_as_radian<1>(point);
 
         calculation_type const buffer_distance = distance_strategy.apply(point, point,
                         strategy::buffer::buffer_side_left);
 
-        typedef typename FormulaPolicy::template direct
+        using direct_t = typename FormulaPolicy::template direct
             <
                 calculation_type, true, false, false, false
-            > direct_t;
+            >;
 
         calculation_type const two_pi = geometry::math::two_pi<calculation_type>();
         calculation_type const pi = geometry::math::pi<calculation_type>();
 
         calculation_type const diff = two_pi / calculation_type(m_count);
-        // TODO: after calculation of some angles is corrected,
-        // we can start at 0.0
-        calculation_type angle = 0.001;
+        calculation_type angle = -pi;
 
         for (std::size_t i = 0; i < m_count; i++, angle += diff)
         {
-            if (angle > pi)
-            {
-                angle -= two_pi;
-            }
-
-            typename direct_t::result_type
-                dir_r = direct_t::apply(get_as_radian<0>(point), get_as_radian<1>(point),
-                                        buffer_distance, angle,
+            // If angle is zero, shift angle a tiny bit to avoid spikes.
```

Comment:
is this some fix on buffer point geographic strategy to avoid spikes or is another issue introduced by the new buffer line strategies?

Reply:
This was already the case in earlier code. It's now less but not yet 100% disappeared.
We have to come back to it later. If I encounter it, I will create an issue.